index.html

```diff
@@ -0,0 +1,336 @@
+        </ul>
+      </div>
+
+      <div class="desk-card-w1">
+        <h2 class="desk-prof-art-w">Website Protfolio </h2>
+        <p class="desk-w">A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+          the industry's standard</p>
+        <ul class="desk-skils-d">
+          <li class="desk-button-d button-d">html</li>
+          <li class="desk-skil button-d">bootstrap</li>
+          <li class="desk-skil button-d">Ruby</li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="card456">
+      <div class="desk-card-d1">
+        <h2 class="desk-prof-art-dash">Profesional Art Printing Data More</h2>
+        <p class="desk-p22-d">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+          been the industry's standard</p>
+        <ul class="desk-skils-d">
+          <li class="desk-skil button-d">html</li>
+          <li class="desk-skil button-d">bootstrap</li>
+          <li class="desk-skil button-d">Ruby</li>
+        </ul>
+      </div>
+
+      <div class="desk-card-d2">
+        <h2 class="desk-prof-art-dash">Data Dashboard Healthcare</h2>
+        <p class="desk-p22-d">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+          been the industry's standard</p>
+        <ul class="desk-skils-d">
+          <li class="desk-skil button-d">html</li>
+          <li class="desk-skil button-d">bootstrap</li>
+          <li class="desk-skil button-d">Ruby</li>
+        </ul>
+
+      </div>
+
+      <div class="desk-card-w1">
+        <h2 class="desk-prof-art-w">Website Protfolio </h2>
+        <p class="desk-w">A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+          the industry's standard</p>
+        <ul class="desk-skils-d">
+          <li class="desk-button-d button-d">html</li>
+          <li class="desk-skil button-d">bootstrap</li>
+          <li class="desk-skil button-d">Ruby</li>
+        </ul>
+      </div>
+    </div>
+  </section>
+
+  <section class="section-3">
+    <div class="about-me">
+      <h1 class="h31">About me</h1>
+      <p class="p31-desk">Hello I’m a software developer! I can help you build a product ,feature or
+        website Look through some of my work and experience! If you like what you see and have a project you need coded,
+        don’t hestiate to contact me.</p>
+      <a class="get-myresume button" href="">Get My Resume</a>
+    </div>
+    <hr class="line-3">
+    <div class="about-me-desk">
+      <div class="story-abt-me">
+        <h1 class="h31-desk">About me</h1>
+        <p class="p31-desk">Hello I’m a software developer! I can help you build a product ,feature or
+          website Look through some of my work and experience! If you like what you see and have a project you need
+          coded, don’t hestiate to contact me.</p>
+        <a class="get-myresume button" href="">Get My Resume</a>
+      </div>
```

Comment:
- Great job implementing this.
- I'd like to kindly suggest that the project requires us to add a hover effect on the button. Can we kindly implement this effect on all buttons where necessary? 🌸 

Here is what I mean 👇🏽 

| Button Default color  |Button color on hOVER |
| :---:   | :---: | 
| <img width="196" alt="Screenshot 2023-05-04 at 11 53 20" src="https://user-images.githubusercontent.com/99246359/236272618-6cc4626a-7bb0-45a0-a33b-fa8d94555bad.png"> |<img width="167" alt="Screenshot 2023-05-04 at 11 39 03" src="https://user-images.githubusercontent.com/99246359/236272674-fb672914-e88a-463d-be17-fdfb44182e3b.png">  |